### PR TITLE
Add CREED space

### PR DESCRIPTION
**Pull request by creedev**

### CODE REVIEW

File: spaces/creed/skin.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.name {
+  --primary-color: #384aff;
+  --bg-color: white;
+  --text-color: #586069;
+  --link-color: #111111;
+  --heading-color: #111111;
+  --border-color: #d1d5da;
+  --header-bg: white;
+  --block-bg: transparent;
+}
```
Comment (e-asia) [inline spaces/creed/skin.scss:10]: To use the default theme you will want to delete skin.scss from your space folder.

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```
Comment (e-asia) [inline yarn.lock:4]: This file should not be in your space folder.

**Comment by creedev** [comment]
@e-asia  Can you please review once now. Thanks.

File: yarn.lock (hunk 2 above)
Comment (creedev) [reply yarn.lock]: Can you please review once now. Thanks.

File: spaces/creed/skin.scss (hunk 1 above)
Comment (creedev) [reply spaces/creed/skin.scss]: Can you please review once now. Thanks.

**Comment by creedev** [comment]
https://github.com/bonustrack/snapshot-spaces/pull/62
I cleaned up the code and PR again.
#60 PR is deleted

**Pull request closed without merging**